### ISSUE TITLE
Geogram link fails

### ISSUE THREAD

**Issue by Bleyddyn** [alicevision/AliceVision#1128]
Cmake is failing for me because it can't find Geogram, but I'm having a hard time finding the library.

In INSTALL.md, the link for downloading Geogram, https://gforge.inria.fr/frs/?group_id=5833,  gives a 403 Forbidden error.

And I don't see any Debian packages that look like they might be pre-built versions.

Any help would be appreciated.

**Comment by Bleyddyn** [comment]
Thanks. I did eventually get AliceVision to build, although I later figured out I could just download the Meshroom binary.

I did have to install quite a few dependencies that I didn't see mentioned in any of the documentation. I'm including what I installed, in case it helps someone else.

sudo apt install libopenexr-dev
sudo apt install libopenimageio-dev
sudo apt install libeigen3-dev
sudo apt install libceres-dev
sudo apt install libassimp-dev
https://github.com/alicevision/geogram.git
sudo apt install libxinerama-dev
sudo apt install libxcursor-dev
sudo apt install libboost-atomic-dev libboost-container-dev libboost-date-time-dev libboost-filesystem-dev libboost-graph-dev libboost-log-dev libboost-program-options-dev libboost-regex-dev libboost-serialization-dev libboost-system-dev libboost-thread-dev libboost-timer-dev
